projects/noloan/app/src/main/java/hasadna/noloan/lawsuit/fragments/LawsuitFormFragment.java

```diff
@@ -28,6 +30,7 @@
   EditText receivedDate;
   TextInputEditText userId;
   TextView idValidationError;
+  Button searchCompany;
```

Comment:
`searchCompany` button is only used in the `OnCreateView` function. It doesn't need to be a class field.
Also true to other views, such as `confirmForm`.

Reply:
:+1: